test/initialize.jl

```diff
@@ -59,6 +59,36 @@ end
     """
 end
 
+
+@testset "Initialize from Vorticity IC" begin
+
+    # BAROTROPIC MODEL
+    progn, diagn, model = initialize_speedy(initial_conditions=:rest,model=:barotropic)
+
+    ic = [layer.leapfrog[1].vor for layer in progn.layers]
+    progn, diagn, model = initialize_speedy(initial_conditions=ic,model=:barotropic)
+    for layers in progn.layers
+        for leapfrog in layers.leapfrog
+            @test all(leapfrog.vor .== 0)
```

Comment:
Given the open broadcasting issue related to the comment above, note that this is a redundant test: Replacing zeros with zeros would pass even if nothing is actually done.

Reply:
Same here as well, that's essentially the same test as already done for the other initialisations

Reply:
Haha, boooh past me setting up shitty tests 😆